pkg/provider/data_source_cluster.go

```diff
@@ -288,9 +290,32 @@ func (c *clusterDataSource) Schema(ctx context.Context, req datasource.SchemaReq
 				MarkdownDescription: "Earliest backup recover time.",
 				Computed:            true,
 			},
+			"maintenance_window": schema.SingleNestedAttribute{
+				Computed:            true,
+				MarkdownDescription: "Custom maintenance window.",
+				Attributes: map[string]schema.Attribute{
+					"is_enabled": schema.BoolAttribute{
+						MarkdownDescription: "Is maintenance window enabled.",
+						Computed:            true,
+					},
+					"start_day": schema.Float64Attribute{
+						MarkdownDescription: "Start day.",
+						Computed:            true,
+						Validators: []validator.Float64{
+							startDayValidator(),
+						},
+					},
+					"start_time": schema.StringAttribute{
+						MarkdownDescription: "Start time.",
```

Comment:
```suggestion
						MarkdownDescription: "Start time. \"hh:mm\", for example: \"23:59\".",
```
It would be great if you can update this in other recurrences of this start_time.